src/test/java/uk/gov/justice/digital/delius/integration/secure/UserAPITest.java

```diff
@@ -62,18 +63,19 @@ public void retrieveUserDetailsForMultipleUsers() throws JsonProcessingException
                 .statusCode(200)
                 .extract()
                 .body()
-                .as(JsonNode.class);
+                .as(UserDetails[].class);
```

Comment:
would be a much better test if we actually converted to a `String.class` and then compared the json instead.  That way you could compare all the data rather than just a subset of the fields.

Reply:
Good point, agreed and done